style.css

```diff
@@ -3,6 +3,51 @@
   padding: 0;
 }
 
+#overlay {
+  display: none;
+  flex-direction: column;
+  background-color: #6070ff;
+  opacity: 0.9;
+  -webkit-backdrop-filter: blur(50%);
+  backdrop-filter: blur(50%);
+  position: fixed;
+  height: 100%;
+  width: 100%;
+}
+
+#close-container {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+}
+
+#closer {
+  padding: 18px;
+}
```

Comment:
- [ ] I think you should adjust this icon once more. Your icon is more closer to the top right end as compared to the one in the Figma design. You shall Kindly check the screenshots below and make the necessary adjustments.

| Yours | Figma |
| ---    | ---   | 
| ![yours](https://user-images.githubusercontent.com/63087491/188518083-510c0fa7-4581-4d0f-a59d-891f4fa62040.png) | ![Screenshot from 2022-09-05 23-08-00](https://user-images.githubusercontent.com/63087491/188518074-ee2a52c8-b915-466a-904c-05d16017ac1f.png) |